src/Tickets/Commerce/Admin/Orders_Page.php

```diff
@@ -164,4 +180,27 @@ public function is_admin_orders_page() {
 
 		return true;
 	}
+
+	/**
+	 * Checks if the current screen is the orders page and the Tickets Events Calendar is not active.
+	 *
+	 * @since TBD
+	 *
+	 * @return bool
+	 */
+	public function is_admin_orders_page_and_no_TEC() { // phpcs:ignore WordPress.NamingConventions.ValidFunctionName.MethodNameInvalid
+		if ( ! $this->is_admin_orders_page() ) {
+			return false;
+		}
+
+		try {
+			tribe( 'tec.main' );
```

Comment:
```suggestion
			tribe( 'tec.main' );
```
Not a huge fan of using a try catch to validate TEC is installed. I'd prefer looking for a TEC supported constant. And not touching class_exists() checks either, that can be troublesome.

Thoughts?

Reply:
Yea i think constant would definitely be faster and cleaner. Bad choice evaluation through try-catch for sure looking at it now.

Thanks @stratease !